lib/tasks/state_file.rake

```diff
@@ -21,4 +21,39 @@ namespace :state_file do
 
     StateFile::SendReminderApologyService.run
   end
+
+  task backfill_intake_submission_pdfs: :environment do
+    batch_size = 5
+    intake_types = StateFile::StateInformationService.state_intake_classes
+    intake_types.find do |intake_type|
+      sql = <<~SQL
+        SELECT #{intake_type.table_name}.id 
+        FROM #{intake_type.table_name}
+        INNER JOIN efile_submissions ON (
+          efile_submissions.data_source_type = '#{intake_type.name}'
+          AND efile_submissions.data_source_id = #{intake_type.table_name}.id
+        ) INNER JOIN efile_submission_transitions ON (
+          efile_submission_transitions.efile_submission_id = efile_submissions.id
+          AND efile_submission_transitions.to_state not in ('new', 'preparing', 'bundling', 'queued')
+          AND efile_submission_transitions.most_recent = true
+        ) LEFT JOIN active_storage_attachments ON (
+          active_storage_attachments.record_type = '#{intake_type.name}'
+          AND active_storage_attachments.record_id = #{intake_type.table_name}.id
+          AND active_storage_attachments.name = 'submission_pdf'
+        ) WHERE active_storage_attachments.id IS NULL
```

Comment:
Have you run this on production to see how many records are affected? And could you add that info in the testing section of the PR summary?

Reply:
NY: 12952
AZ: 2138